lang/quiterss_ar.ts

```diff
@@ -527,7 +529,7 @@
     </message>
     <message>
         <location filename="../src/cleanupwizard.cpp" line="361"/>
-        <source>Totally remove records that had marked 'deleted' from DB.
+        <source>Totally remove records that had marked &apos;deleted&apos; from DB.
```

Comment:
Probably more automatic changes done by your IDE which bloat up the diff. Looks like all translation files are affected.

Reply:
I think so saddly.